omnibus/package-scripts/ddot-deb/prerm

```diff
@@ -0,0 +1,25 @@
+#!/bin/sh
+##########################################################################
+#             DO NOT EDIT THIS SCRIPT DIRECTLY.                          #
+#                                                                        #
+# The installation logic is handled by the installer in the following    #
+# file: pkg/fleet/installer/packages/datadog_agent_ddot_linux.go         #
+#                                                                        #
+##########################################################################
+
+INSTALLER_DEB=/opt/datadog-agent/embedded/bin/installer
+INSTALLER_OCI=/opt/datadog-package/datadog-agent-ddot/stable/embedded/bin/installer
```

Comment:
The installer lives in the agent

```suggestion
INSTALLER_OCI=/opt/datadog-package/datadog-agent/stable/embedded/bin/installer
```

Reply:
As written in another comment, we could simply use `datadog-installer` if the we know the agent is always uninstalled __after__ this package.